app/actions/index.js

```diff
@@ -4,7 +4,7 @@ import {
   fetchCatsFromFb,
   saveProfileToFb
 } from '../modules/firebase-db';
-
+import {browserHistory} from 'react-router';
```

Comment:
Observe how the style guide is applied to the rest of the codebase, and apply that here.